services/tenant-management-service/src/models/tenant-config.model.ts

```diff
@@ -0,0 +1,47 @@
+import {model, property, belongsTo} from '@loopback/repository';
+import {UserModifiableEntity} from '@sourceloop/core';
+import {Tenant} from './tenant.model';
+import {ConfigValue} from '../providers/idp';
+
+@model({
+  name: 'tenant_configs',
+  description: 'tenant_configs to save any tenant specific data related to idP',
+})
+export class TenantConfig extends UserModifiableEntity {
+  @property({
+    type: 'string',
+    id: true,
+    generated: true,
+  })
+  id: string;
+
+  @property({
+    type: 'string',
+    required: true,
+    name: 'config_key',
+  })
+  configKey: string;
+
+  @property({
+    type: 'object',
+    required: true,
+    name: 'config_value',
+  })
+  configValue: ConfigValue;
+
```

Comment:
why is the configValue type defined like this 
it not specific to just auth0 and organizations 
its a generic table that can be used for anything in future
its just a json value https://github.com/sourcefuse/loopback4-microservice-catalog/blob/master/services/authentication-service/src/models/tenant-config.model.ts